identity_jose/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "identity_jose"
+version = "0.7.0-alpha.4"
+authors = ["IOTA Stiftung"]
+edition = "2021"
+homepage = "https://www.iota.org"
+keywords = ["iota", "tangle", "identity", "jose", "jwa", "jwe", "jwk", "jwm", "jws", "jwt"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/iotaledger/identity.rs"
+description = "A library for JOSE (JSON Object Signing and Encryption)"
+
+[dependencies]
+base64 = { version = "0.21.0", default-features = false, features = ["std"] }
+iota-crypto = { version = "0.15.3", default-features = false, features = ["std", "sha"] }
+serde = { version = "1.0", default-features = false, features = ["std", "derive"] }
+serde_json = { version = "1.0", default-features = false, features = ["std"] }
```

Comment:
Nit: Perhaps these crates should also go in `workspace.dependencies` since they seem to be fairly common (either as essential dependencies or at least as dev-dependencies).

Reply:
Yeah, that makes sense. Once I've merged this PR into main I will make a small PR that hoists `serde_json` into the top-level `Cargo.toml`.